# translation of pl.po to Polish
# Piotr Drąg <piotrdrag@gmail.com>, 2008-2015.
#
msgid ""
msgstr ""
"Project-Id-Version: liveusb-creator\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-11-02 19:40+0100\n"
"PO-Revision-Date: 2015-11-02 20:28+0100\n"
"Last-Translator: Piotr Drąg <piotrdrag@gmail.com>\n"
"Language-Team: Polish <trans-pl@lists.fedoraproject.org>\n"
"Language: pl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
"|| n%100>=20) ? 1 : 2);\n"

#: liveusb/components/DownloadDialog.qml:9
msgctxt ""
msgid "Write %1 to USB"
msgstr "Zapisz %1 na urządzenie USB"

#: liveusb/components/DownloadDialog.qml:250
msgctxt ""
msgid "There are no portable drives connected"
msgstr "Nie podłączono żadnych przenośnych dysków"

#: liveusb/components/DownloadDialog.qml:290
msgctxt ""
msgid "Options"
msgstr "Opcje"

#: liveusb/components/DownloadDialog.qml:312
msgctxt ""
msgid "Cancel"
msgstr "Anuluj"

#: liveusb/components/DownloadDialog.qml:335
msgctxt ""
msgid "Close"
msgstr "Zamknij"

#: liveusb/components/DownloadDialog.qml:335
msgctxt ""
msgid "Are you sure?"
msgstr "Na pewno?"

#: liveusb/components/DownloadDialog.qml:335
msgctxt ""
msgid "Write to disk"
msgstr "Zapisz na dysk"

#: liveusb/components/DownloadDialog.qml:437
msgctxt ""
msgid "Write the image immediately when the download is finished"
msgstr "Zapisz obraz od razu po ukończeniu pobierania"

#: liveusb/components/ImageDetails.qml:46
msgctxt ""
msgid "Create Live USB"
msgstr "Utwórz LiveUSB"

#: liveusb/components/ImageDetails.qml:144
msgctxt ""
msgid "Version %1"
msgstr "Wersja %1"

#: liveusb/components/ImageDetails.qml:145
msgctxt ""
msgid ", released on %1"
msgstr ", wydana w dniu %1"

#: liveusb/components/ImageDetails.qml:156
msgctxt ""
msgid "Select Live ISO"
msgstr "Wybierz plik ISO obrazu Live"

#: liveusb/components/ImageDetails.qml:164
msgctxt ""
msgid "Selected:"
msgstr "Wybrano:"

#: liveusb/components/ImageDetails.qml:164
msgctxt ""
msgid "None"
msgstr "Brak"

#: liveusb/components/ImageDetails.qml:226
msgctxt ""
msgid "Image files (*.iso)"
msgstr "Pliki obrazów (*.iso)"

#: liveusb/components/ImageDetails.qml:226
msgctxt ""
msgid "All files (*)"
msgstr "Wszystkie pliki (*)"

#: liveusb/components/ImageList.qml:85
msgctxt ""
msgid "Find an operating system image"
msgstr "Znajdź obraz systemu operacyjnego"

#: liveusb/components/ImageList.qml:212
msgctxt ""
msgid "Display additional Fedora flavors"
msgstr "Wyświetl dodatkowe rodzaje Fedory"

#: liveusb/creator.py:146
msgid "Setting up OLPC boot file..."
msgstr "Ustawianie pliku startowego OLPC…"

#: liveusb/creator.py:207
#, python-format
msgid ""
"There was a problem executing the following command: %r\n"
"%r\n"
"A more detailed error log has been written to '%r'"
msgstr ""
"Wystąpił problem podczas wykonywania następującego polecenia: %r\n"
"%r\n"
"Bardziej szczegółowy dziennik błędu został zapisany do pliku „%r”"

#: liveusb/creator.py:224
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Sprawdzanie sumy kontrolnej SHA1 obrazu Live CD…"

#: liveusb/creator.py:228
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Sprawdzanie sumy kontrolnej SHA256 obrazu Live CD…"

#: liveusb/creator.py:244
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr ""
"Błąd: suma kontrolna SHA1 obrazu Live CD jest nieprawidłowa. Można uruchomić "
"ten program za pomocą parametru „--noverify”, aby obejść sprawdzanie "
"poprawności."

#: liveusb/creator.py:250
msgid "Unknown ISO, skipping checksum verification"
msgstr "Nieznany plik ISO, pomijanie sprawdzania sum kontrolnych"

#: liveusb/creator.py:261
#, python-format
msgid ""
"There is not enough free space on the selected device.\n"
"Required: %s. Free: %s."
msgstr ""
"Za mało wolnego miejsca na wybranym urządzeniu.\n"
"Wymagane: %s. Wolne: %s."

#: liveusb/creator.py:267
msgid "Creating"
msgstr "Tworzenie"

#: liveusb/creator.py:268
msgid "persistent overlay"
msgstr "warstwa trwałości"

#: liveusb/creator.py:330
#, python-format
msgid "Unable to copy %r to %r: %r"
msgstr "Nie można skopiować %r do %r: %r"

#: liveusb/creator.py:338 liveusb/creator.py:1336
msgid "Removing"
msgstr "Usuwanie"

#: liveusb/creator.py:344
msgid "Removing existing Live OS"
msgstr "Usuwanie istniejącego systemu operacyjnego Live"

#: liveusb/creator.py:350
msgid "Deleting "
msgstr "Usuwanie "

#: liveusb/creator.py:357
#, python-format
msgid "Unable to delete %r: %r"
msgstr "Nie można usunąć %r: %r"

#: liveusb/creator.py:361
#, python-format
msgid "Unable to remove previous LiveOS: %r"
msgstr "Nie można usunąć poprzedniego systemu operacyjnego Live: %r"

#: liveusb/creator.py:379
#, python-format
msgid "Cannot find path %r"
msgstr "Nie można odnaleźć ścieżki %r"

#: liveusb/creator.py:410
#, python-format
msgid "Cannot find device %s"
msgstr "Nie można odnaleźć urządzenia %s"

#: liveusb/creator.py:461
msgid "Overwriting device with live image"
msgstr "Zastępowanie zawartości urządzenia obrazem Live"

#: liveusb/creator.py:468
msgid "Running"
msgstr "Wykonywanie"

#: liveusb/creator.py:517
msgid ""
"You are using an old version of syslinux-extlinux that does not support the "
"ext4 filesystem"
msgstr ""
"Używana jest stara wersja programu „syslinux-extlinux”, która nie obsługuje "
"systemu plików ext4"

#: liveusb/creator.py:741 liveusb/creator.py:803
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr ""
"Nieznany system plików. Urządzenie może wymagać ponownego sformatowania."

#: liveusb/creator.py:744 liveusb/creator.py:806
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Nieobsługiwany system plików: %s"

#: liveusb/creator.py:757 liveusb/creator.py:774
#, python-format
msgid "Unknown dbus exception while trying to mount device: %s"
msgstr ""
"Nieznany wyjątek usługi D-Bus podczas próby zamontowania urządzenia: %s"

#: liveusb/creator.py:777
#, python-format
msgid "Unable to mount device: %r"
msgstr "Nie można zamontować urządzenia: %r"

#: liveusb/creator.py:780
msgid "No mount points found after mounting attempt"
msgstr "Nie odnaleziono punktów montowania po próbie zamontowania"

#: liveusb/creator.py:788
#, python-format
msgid "Using existing mount: %s"
msgstr "Używanie istniejącego punktu montowania: %s"

#: liveusb/creator.py:800 liveusb/creator.py:1273
msgid "Verifying filesystem..."
msgstr "Sprawdzanie poprawności systemu plików…"

#: liveusb/creator.py:809
#, python-format
msgid "Setting %s label to %s"
msgstr "Ustawianie etykiety %s na %s"

#: liveusb/creator.py:823
#, python-format
msgid "Unable to change volume label: %r"
msgstr "Nie można zmienić etykiety woluminu: %r"

#: liveusb/creator.py:827 liveusb/creator.py:1305
msgid "Extracting live image to USB device..."
msgstr "Rozpakowywanie obrazu Live na urządzenie USB…"

#: liveusb/creator.py:833
msgid "Unable to find LiveOS on ISO"
msgstr "Nie można odnaleźć systemu operacyjnego Live w pliku ISO"

#: liveusb/creator.py:845 liveusb/creator.py:1313
msgid "Wrote to device at"
msgstr "Zapisano do urządzenia w"

#: liveusb/creator.py:873
msgid "Installing bootloader..."
msgstr "Instalowanie programu startowego…"

#: liveusb/creator.py:917
#, python-format
msgid "Cannot find device: %s"
msgstr "Nie można odnaleźć urządzenia: %s"

#: liveusb/creator.py:941
msgid "Verifying ISO MD5 checksum"
msgstr "Sprawdzanie sumy kontrolnej MD5 pliku ISO"

#: liveusb/creator.py:946
msgid "ISO MD5 checksum verification failed"
msgstr "Sprawdzenie sumy kontrolnej MD5 pliku ISO się nie powiodło"

#: liveusb/creator.py:948
msgid "ISO MD5 checksum passed"
msgstr "Sprawdzenie sumy kontrolnej MD5 pliku ISO się powiodło"

#: liveusb/creator.py:989
msgid "Unable to get disk partitions"
msgstr "Nie można odnaleźć partycji dysku"

#: liveusb/creator.py:993
#, python-format
msgid "%s already bootable"
msgstr "%s jest już startowa"

#: liveusb/creator.py:1013
msgid "Unable to find partition"
msgstr "Nie można odnaleźć partycji"

#: liveusb/creator.py:1082
msgid "Resetting Master Boot Record"
msgstr "Przywracanie głównego sektora startowego"

#: liveusb/creator.py:1085
msgid "Unable to reset MBR.  You may not have the `syslinux` package installed"
msgstr "Nie można przywrócić MBR. Brak zainstalowanego pakietu „syslinux”"

#: liveusb/creator.py:1088
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Dysk jest urządzeniem zwrotnym, pomijanie przywracania MBR"

#: liveusb/creator.py:1092 liveusb/creator.py:1471
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Obliczanie sumy kontrolnej SHA1 pliku %s"

#: liveusb/creator.py:1222
#, python-format
msgid "Unable to get GetVolumeInformation(%r): %r"
msgstr "Nie można uzyskać informacji o woluminie (%r): %r"

#: liveusb/creator.py:1236
msgid "Error probing device"
msgstr "Błąd podczas wykrywania urządzenia"

#: liveusb/creator.py:1277
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr ""
"Proszę się upewnić, że urządzenie USB jest podłączone i sformatowane w "
"systemie plików FAT"

#: liveusb/creator.py:1280
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr ""
"Nieobsługiwany system plików: %s\n"
"Proszę wykonać kopię zapasową i sformatować urządzenie USB w systemie plików "
"FAT."

#: liveusb/creator.py:1290
#, python-format
msgid "Unable to SetVolumeLabel: %r"
msgstr "Nie można ustawić etykiety woluminu: %r"

#: liveusb/creator.py:1299
#, python-format
msgid "Problem determining free space: %r"
msgstr "Problem podczas ustalania wolnego miejsca: %r"

#: liveusb/creator.py:1319
msgid "Installing bootloader"
msgstr "Instalowanie programu startowego"

#: liveusb/creator.py:1359
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr ""
"Nie można uzyskać Win32_LogicalDisk. Zapytanie win32com nie zwróciło żadnych "
"wyników"

#: liveusb/creator.py:1380
#, python-format
msgid "Found UUID %s for %s"
msgstr "Odnaleziono UUID %s dla %s"

#: liveusb/creator.py:1383
#, python-format
msgid "Exception while fetching UUID: %r"
msgstr "Wyjątek podczas pobierania UUID: %r"

#: liveusb/creator.py:1391
#, python-format
msgid "Max size of %s: %d"
msgstr "Maksymalny rozmiar %s: %d"

#: liveusb/creator.py:1394
#, python-format
msgid "Error getting drive size: %r"
msgstr "Błąd podczas pobierania rozmiaru dysku: %r"

#: liveusb/creator.py:1412
msgid "Cannot find"
msgstr "Nie można odnaleźć"

#: liveusb/creator.py:1413
msgid ""
"Make sure to extract the entire liveusb-creator zip file before running this "
"program."
msgstr ""
"Proszę się upewnić, że rozpakowano cały plik ZIP programu liveusb-creator "
"przed jego uruchomieniem."

#: liveusb/creator.py:1458
#, python-format
msgid "Unable to detect proxy settings: %r"
msgstr "Nie można wykryć ustawień pośrednika: %r"

#: liveusb/creator.py:1459
#, python-format
msgid "Using proxies: %r"
msgstr "Używanie pośredników: %r"

#: liveusb/gui.py:243
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr ""
"Błąd: nie można ustawić etykiety lub uzyskać UUID urządzenia. Nie można "
"kontynuować."

#: liveusb/gui.py:271
msgid "Checking the source image"
msgstr "Sprawdzanie obrazu źródłowego"

#: liveusb/gui.py:287
msgid "Unpacking the image"
msgstr "Rozpakowywanie obrazu"

#: liveusb/gui.py:299
msgid "Writing the data"
msgstr "Zapisywanie danych"

#: liveusb/gui.py:305
msgid "Checking the written data"
msgstr "Sprawdzanie zapisanych danych"

#: liveusb/gui.py:316
msgid "Finished!"
msgstr "Ukończono."

#: liveusb/gui.py:502
msgid ""
"You are about to perform a destructive install. This will erase all data and "
"partitions on your USB drive"
msgstr ""
"Za chwilę zostanie wykonana instalacja, która usunie wszystkie dane i "
"partycje na urządzeniu USB"

#: liveusb/gui.py:505
msgid ""
"The Master Boot Record on your device is blank. Writing the image will reset "
"the MBR on this device"
msgstr ""
"Główny sektor startowy na urządzeniu jest pusty. Zapisanie obrazu przywróci "
"MBR na tym urządzeniu"

#: liveusb/gui.py:507
msgid ""
"The Master Boot Record on your device does not match your system's syslinux "
"MBR.\n"
"If you have trouble booting it, try setting the \"Reset the MBR\" advanced "
"option."
msgstr ""
"Główny sektor startowy na urządzeniu nie zgadza się z MBR syslinuksa "
"systemu.\n"
"Jeśli podczas uruchamiania tego dysku wystąpił błąd, proszę spróbować "
"ustawić zaawansowaną opcję „Przywróć MBR”."

#: liveusb/gui.py:515
msgid "Unable to mount device"
msgstr "Nie można zamontować urządzenia"

#: liveusb/gui.py:518
msgid ""
"Your device already contains a live OS. If you continue, it will be "
"overwritten."
msgstr ""
"Urządzenie zawiera już system operacyjny Live. W przypadku kontynuacji "
"zostanie on zastąpiony."

#: liveusb/gui.py:621
msgid "Starting"
msgstr "Rozpoczynanie"

#: liveusb/gui.py:623
msgid "Downloading"
msgstr "Pobieranie"

#: liveusb/gui.py:625
msgid "Error"
msgstr "Błąd"

#: liveusb/gui.py:627
msgid "Ready to write"
msgstr "Gotowe do zapisania"

#: liveusb/gui.py:631
msgid "Finished"
msgstr "Ukończono"

#: liveusb/gui.py:789
msgid ""
"Use <b>dd</b> to write the image - this will erase everything on your "
"portable drive"
msgstr ""
"Użyj polecenia <b>dd</b> do zapisania obrazu — usunie to wszystkie dane na "
"urządzeniu USB"

#: liveusb/gui.py:790
msgid "Reset the MBR (Master Boot Record)"
msgstr "Przywróć MBR (Główny sektor startowy)"

#: liveusb/liveusb.qml:16
msgctxt ""
msgid "Fedora LiveUSB Creator"
msgstr "Tworzenie LiveUSB Fedory"

#: liveusb/releases.py:239
msgid "Custom OS..."
msgstr "Inny system operacyjny…"

#: liveusb/releases.py:240
msgid ""
"<p>Here you can choose a OS image from your hard drive to be written to your "
"flash disk</p><p>Currently it is only supported to write raw disk images (."
"iso or .bin)</p>"
msgstr ""
"<p>Tutaj można wybrać obraz ISO z dysku twardego.</p><p>Obecnie obsługiwane "
"są tylko surowe obrazy (.iso lub .bin)</p>"

#: liveusb/releases.py:243
msgid "Pick a file from your drive(s)"
msgstr "Wybierz plik z dysku"

#: liveusb/urlgrabber/grabber.py:693 liveusb/urlgrabber/grabber.py:1172
#, python-format
msgid "Bad URL: %s"
msgstr "Błędny adres URL: %s"

#: liveusb/urlgrabber/grabber.py:781
#, python-format
msgid "Illegal reget mode: %s"
msgstr "Niedozwolony tryb „reget”: %s"

#: liveusb/urlgrabber/grabber.py:906
#, python-format
msgid "Local file does not exist: %s"
msgstr "Lokalny plik nie istnieje: %s"

#: liveusb/urlgrabber/grabber.py:909
#, python-format
msgid "Not a normal file: %s"
msgstr "Nie jest zwykłym plikiem: %s"

#: liveusb/urlgrabber/grabber.py:967
#, python-format
msgid "Exceeded limit (%i): %s"
msgstr "Przekroczono ograniczenie (%i): %s"

#: liveusb/urlgrabber/grabber.py:1150
msgid "Byte range requested but range support unavailable"
msgstr "Zażądano zakresu bajtów, ale brak obsługi zakresów"

#: liveusb/urlgrabber/grabber.py:1183 liveusb/urlgrabber/grabber.py:1262
#, python-format
msgid "Timeout: %s"
msgstr "Czas oczekiwania: %s"

#: liveusb/urlgrabber/grabber.py:1185 liveusb/urlgrabber/grabber.py:1264
#, python-format
msgid "IOError: %s"
msgstr "Błąd wejścia/wyjścia: %s"

#: liveusb/urlgrabber/grabber.py:1187
#, python-format
msgid "OSError: %s"
msgstr "Błąd systemu operacyjnego: %s"

#: liveusb/urlgrabber/grabber.py:1189
#, python-format
msgid "HTTP Exception (%s): %s"
msgstr "Wyjątek HTTP (%s): %s"

#: liveusb/urlgrabber/grabber.py:1260
#, python-format
msgid "Socket Error: %s"
msgstr "Błąd gniazda: %s"

#: liveusb/urlgrabber/grabber.py:1332
#, python-format
msgid "Bad proxy URL: %s"
msgstr "Błędny adres URL pośrednika: %s"

#: liveusb/urlgrabber/mirror.py:287
msgid "No more mirrors to try."
msgstr "Nie ma więcej serwerów lustrzanych do wypróbowania."
